src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.Square;
+import core.basesyntax.figuresupplier.FigureSupplier;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier1 = new FigureSupplier();
+        Drawable circle = new Circle("red", 21);
+        Drawable rectangle = new Rectangle("Yellow", 21, 20);
+        Drawable square = new Square("Black", 17);
+
+        Drawable[] qwert = new Drawable[]{figureSupplier1.getRandomFigure(),
+                figureSupplier1.getRandomFigure(), figureSupplier1.getRandomFigure(),
+                circle, rectangle, square};
+
+        for (Drawable qw : qwert) {
+            qw.draw();
+        }
```

Comment:
```suggestion
        FigureSupplier figureSupplier = new FigureSupplier();
        Figure[] figures = new Figure[NUMBER_OF_FIGURES]
        for (int i = 0; i < FIGURE_NUMBER; i++) {
            if (i < NUMBER_OF_FIGURES/ 2) {
                figures[i] = figureSupplier.getRandomFIgure()
            } else {
                figures[i] = figureSupplier.getDefaultFIgure()
            }
            figures[i].draw();
        }
```
![image](https://user-images.githubusercontent.com/81781479/193444700-a417b892-64f3-451a-91ce-6a8a540bd8e9.png)
using abstraction is not a bad practice, but we dont need to use the most abstract abstraction

using Drawable would be good if the task was to create different classes like Picture (and subclasses Portrait, Landscape etc), Figure (and subclasses Cricle etc) and a lot more

but here your task was only to create Figure - that is the main abstract class, we should work with it